executor/ddl.go

```diff
@@ -118,7 +114,14 @@ func (e *DDLExec) Next(ctx context.Context, req *chunk.Chunk) (err error) {
 
 	}
 	if err != nil {
-		return e.toErr(err)
+		// If owner return ErrTableNotExists error when running this DDL, It maybe cause by schema changed,
+		// otherwise, ErrTableNotExists could return before put this DDL job to job queue.
+		if (e.ctx.GetSessionVars().StmtCtx.IsDDLJobInQueue && infoschema.ErrTableNotExists.Equal(err)) ||
```

Comment:
How about we execute drop table at the same table in parallel? Add the two DDL statements all pass the check, and generate the DDL jobs. Then the second job needs to return `ErrTableNotExists`, but in this logical, it seems to return  `Information schema is changed`.

Reply:
yep...